snapshot/lib/snapshot/collector.rb

```diff
@@ -7,8 +7,15 @@ def self.fetch_screenshots(output, dir_name, device_type, launch_arguments_index
       containing = File.join(TestCommandGenerator.derived_data_path, "Logs", "Test")
       attachments_path = File.join(containing, "Attachments")
 
-      to_store = attachments(containing)
-      matches = output.scan(/snapshot: (.*)/)
+      language_folder = File.join(Snapshot.config[:output_directory], dir_name)
+      FileUtils.mkdir_p(language_folder)
+
+      if Snapshot.config[:simultaneous] || Helper.xcode_at_least?(9)
```

Comment:
I don't understand that `||`, what if it's Xcode 8, but `simultaneous` is true? Can this ever be the case?